include/rocksdb/advanced_options.h

```diff
@@ -717,6 +717,50 @@ struct AdvancedColumnFamilyOptions {
   // data is left uncompressed (unless compression is also requested).
   uint64_t sample_for_compression = 0;
 
+  // UNDER CONSTRUCTION -- DO NOT USE
```

Comment:
Maybe return non-ok status in `ValidateOptions()` if application sets these options to non-default value?

Reply:
Hm, I think if we were going to do that, it would be an extremely short-lived check: in the next step, I'm planning to start using these options and setting them in tests. So I do need them to be configurable, they're just not ready for end users yet.